GUI/Types/Renderer/ShaderLoader.cs

```diff
@@ -15,7 +15,7 @@ public class ShaderLoader
     {
         private const string ShaderDirectory = "GUI.Types.Renderer.Shaders.";
         private const int ShaderSeed = 0x13141516;
-        private static Regex RegexInclude = new Regex(@"^#include ""(?<IncludeName>[^""]+)""$", RegexOptions.Multiline);
```

Comment:
Why?

Reply:
`GL.GetShader(vertexShader, ShaderParameter.CompileStatus, out var shaderStatus);`
Didn't you get the exception? I come to this exception and think the $ will not match when the next char is \r\n

Reply:
I added an optional `\r` there.

Can you test latest master for both that and the exporting code? I changed it to use same code as the renderer so it doesn't need to be updated in two places.

Reply:
all tested ok